res/linux/org.mixxx.Mixxx.metainfo.xml

```diff
@@ -98,7 +98,7 @@
   Do not edit it manually.
   -->
   <releases>
-    <release version="2.3.1" date="2021-09-29" type="stable">
+    <release version="2.3.1" type="stable" date="2021-09-29" timestamp="1632866400">
```

Comment:
1632866400 converts to 2021-09-28 22:00:00 not 2021-09-29
Can we just drop the redundant date property?

Reply:
I think we should do the opposite and drop the timestamp property.

Reply:
If it is sufficient. I just thought timestamp it is mandatory for flathub. 
> attribute-missing     : <release> has no timestamp

Reply:
Why not just keep both? It's literally one line and since they are autogenerated, they can't get out of sync.